### ISSUE TITLE
NullPointerException on `OpenSSHKeyV1KeyFile.java`

### ISSUE THREAD

**Issue by 134130** [hierynomus/sshj#897]
https://github.com/hierynomus/sshj/blob/f4d34d899d9cf91b545a37d75446110440650970/src/main/java/com/hierynomus/sshj/userauth/keyprovider/OpenSSHKeyV1KeyFile.java#L215

```
java.lang.NullPointerException: Cannot invoke "String.startsWith(String)" because "line" is null
```

**Comment by 134130** [comment]
`reader.readLine()`
> A String containing the contents of the line, not including any line-termination characters, or null if the end of the stream has been reached without reading any characters

**Comment by hierynomus** [comment]
PR for this has been merged in #898
